demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -346,59 +391,188 @@ int main(int argc, char *argv[]) {
                         /** Drawing only objects when > confidence_threshold probability **/
                         std::ostringstream conf;
                         conf << ":" << std::fixed << std::setprecision(3) << confidence;
-                        cv::putText(curr_frame,
+                        cv::putText(requestResult.frame,
                                     (static_cast<size_t>(label) < labels.size() ?
                                     labels[label] : std::string("label #") + std::to_string(label)) + conf.str(),
                                     cv::Point2f(xmin, ymin - 5), cv::FONT_HERSHEY_COMPLEX_SMALL, 1,
                                     cv::Scalar(0, 0, 255));
-                        cv::rectangle(curr_frame, cv::Point2f(xmin, ymin), cv::Point2f(xmax, ymax), cv::Scalar(0, 0, 255));
+                        cv::rectangle(requestResult.frame, cv::Point2f(xmin, ymin), cv::Point2f(xmax, ymax),
+                                      cv::Scalar(0, 0, 255));
                     }
                 }
-            }
 
-            if (!FLAGS_no_show) {
-                cv::imshow("Detection results", curr_frame);
+                presenter.drawGraphs(requestResult.frame);
+
+                std::ostringstream out;
+                out << "FPS: " << std::fixed << std::setprecision(1) << modeInfo[isUserSpecifiedMode].framesCount / 
+                    std::chrono::duration_cast<sec>(std::chrono::high_resolution_clock::now() -
+                                                    modeInfo[isUserSpecifiedMode].lastStartTime).count();
+                cv::putText(requestResult.frame, out.str(), cv::Point2f(10, 25), cv::FONT_HERSHEY_TRIPLEX, 0.6, 
+                            cv::Scalar(0, 0, 255), 1);
+                out.str("");
+                modeInfo[isUserSpecifiedMode].latencySum += std::chrono::duration_cast<sec>(
+                    std::chrono::high_resolution_clock::now() - requestResult.startTime).count();
+                out << "Latency: " << std::fixed << std::setprecision(1) << (modeInfo[isUserSpecifiedMode].latencySum /
+                    modeInfo[isUserSpecifiedMode].framesCount) * 1e3 << " ms";
+                cv::putText(requestResult.frame, out.str(), cv::Point2f(10, 50), cv::FONT_HERSHEY_TRIPLEX, 0.6, 
```

Comment:
I think you should add `put_highlighted_text()` as for `object_detection_demo_yolov3_async.py`.